pkg/rootless/rootless.go

```diff
@@ -9,8 +9,32 @@ import (
 	"github.com/opencontainers/runc/libcontainer/user"
 	spec "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
+	"github.com/syndtr/gocapability/capability"
 )
 
+var (
+	hasCapSysAdminOnce sync.Once
+	hasCapSysAdminRet  bool
+	hasCapSysAdminErr  error
+)
+
+// HasCapSysAdmin returns whether the current process has CAP_SYS_ADMIN.
+func HasCapSysAdmin() (bool, error) {
```

Comment:
I think we need this in storage/pkg/unshare so that buildah could use it.

Reply:
buildah has already the equivalent check, I'll refactor podman and buildah to use the common code

Reply:
https://github.com/containers/storage/pull/877